synapsis/src/main/java/com/soen/synapsis/appuser/AppUser.java

```diff
@@ -118,12 +122,34 @@ public void setAuthProvider(AuthProvider authProvider) {
         this.authProvider = authProvider;
     }
 
-    public Long getCompanyId() {
-        return companyId;
+    public Set<AppUser> getRecruiter() {
+        if(this.getRole() != Role.COMPANY) {
+            throw new IllegalStateException("You must be company to have recruiters.");
+        }
+        return recruiters;
+    }
+
+    public void setRecruiter(AppUser recruiter) {
+        if(this.getRole() != Role.COMPANY) {
+            throw new IllegalStateException("You must be a company to set recruiters.");
+        }
+        if(this.recruiters == null) {
+            this.recruiters = new HashSet<AppUser>();
+        }
+        if(recruiter.getRole() == Role.RECRUITER) {
+            recruiters.add(recruiter);
+        }
+        if(recruiter.getRole() != Role.RECRUITER) {
```

Comment:
I don't understand the point of this, if they are not a recruiter, why would they be in the recruiter list? Also, removals should probably be in a separate function

Reply:
This has been fixed.